### ISSUE TITLE
Highcharts 4.1.8 contents have changed when loaded with bower

### ISSUE THREAD

**Issue by benmccormick** [highcharts/highcharts#4820]
Hi, up until ~today, when installing Highcharts using bower, a highcharts.js file was included at the root of the `bower_components/highcharts` directory.  

a ls of the directory (the result of pulling '4.1.8' using bower) resulted in

```
README.md        adapters        bower.json        highcharts-3d.js    highcharts-3d.src.js    highcharts-more.js    highcharts-more.src.js    highcharts.js        highcharts.src.js    modules            package.json        themes
```

as of today it results in:

```
bower-info.js            changelog-highmaps.htm   generate-changelog.js    invalidations.txt
bower.json               changelog-highstock.htm  grunt-tasks              package.json
changelog-highcharts.htm copy-release.js          gruntfile.js
```

I assume this is a result of `http://www.highcharts.com/component/content/article/2-news/188-highcharts-optimized-for-npm` but I don't understand why it broke previously working past release tags.  This is deeply frustrating as It has broken all of our companies builds, despite the fact we froze our dependencies to point to a specific version.  

I see the bower-info notice, and we'll be migrating to use the new repositories, but in the meantime is there anyway to revert `4.1.8` to its previous tested and stable state?

**Comment by benmccormick** [comment]
Ok I just found this: https://github.com/highcharts/highcharts/issues/4815

Just to clarify this means that you changed the repo that bower was pointing to, breaking any code that was relying on a specific old version staying stable, correct?

**Comment by benmccormick** [comment]
Closed since its a duplicate of #4815, discussing there.
